### PR TITLE
disable Google Translate service during Lighthouse run

### LINKED ISSUE #896: Chrome's Translation prompt kills Lighthouse
Running Lighthouse CLI over adb, I've run into a pretty consistent bug where it dies (possibly times out) when it hits a site that triggers the Google Translate prompt. The bug persists unless you allow or deny automatic translation. 

I wonder if there's a way for us to disable translation somehow.

### PR DESCRIPTION
**Pull request by brendankenny**
fixes #896

### CODE REVIEW

File: lighthouse-core/test/audits/without-javascript-test.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
 const withoutJsAudit = require('../../audits/without-javascript.js');
 const assert = require('assert');
 
-describe.only('Progressive Enhancement: without javascript audit', () => {
+describe('Progressive Enhancement: without javascript audit', () => {
```
Comment (brendankenny) [inline lighthouse-core/test/audits/without-javascript-test.js]: whoops

File: lighthouse-cli/chrome-launcher.ts [hunk 2]
```diff
@@ -53,6 +53,7 @@ class ChromeLauncher {
     const flags = [
       '--remote-debugging-port=9222',
       '--disable-extensions',
+      '--disable-translate',
```
Comment (paulirish) [inline lighthouse-cli/chrome-launcher.ts]: lets add this too: 

* `--disable-default-apps `  "Disables installation of default apps on first run. This is used during automated testing"

Comment (paulirish) [inline lighthouse-cli/chrome-launcher.ts]: there's also 
image
which apparently will flip off some authentication work

**Comment by samccone** [comment]
nice work!

**Pull request merged**